core/src/main/java/com/linecorp/armeria/internal/DefaultAttributeMap.java

```diff
@@ -178,10 +222,13 @@ private static int index(AttributeKey<?> key) {
     @Override
     public String toString() {
         final ToStringHelper helper = MoreObjects.toStringHelper("");
-        for (Iterator<Attribute<?>> i = attrs(); i.hasNext();) {
+        for (final Iterator<Attribute<?>> i = attrs(); i.hasNext();) {
             final Attribute<?> a = i.next();
             helper.add(a.key().name(), a.get());
         }
+        if (parentMap != null) {
+            helper.add(", parent: ", parentMap);
```

Comment:
How about skipping the attributes that exist in own attrs?

Reply:
Oops I made big mistake. Just revert this because it's using `attrs()` which already implemented the feature.